mlvm/mlvm.sh

```diff
@@ -0,0 +1,197 @@
+#!/bin/bash
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# This initialization action will download a set of frequently-used Machine Learning
+# libraries onto a Dataproc cluster, as well as GPU support and connectors for
+# Google Cloud Storage, BigQuery and Spark-Bigquery. 
+
+set -euxo pipefail
+
+readonly JARS_DIR=/usr/lib/spark/jars
+readonly CONNECTORS_DIR=/usr/local/share/google/dataproc/lib
+
+# readonly DEFAULT_INIT_ACTIONS_BUCKET=gs://dataproc-initialization-actions
+# readonly INIT_ACTIONS_BUCKET="$(/usr/share/google/get_metadata_value attributes/INIT_ACTIONS_BUCKET ||
+#   echo ${DEFAULT_INIT_ACTIONS_BUCKET})"
+readonly DEFAULT_INIT_ACTIONS_REPO="https://github.com/GoogleCloudDataproc/initialization-actions.git"
+readonly INIT_ACTIONS_REPO="$(/usr/share/google/get_metadata_value attributes/INIT_ACTIONS_REP ||
+  echo ${DEFAULT_INIT_ACTIONS_REPO})"
```

Comment:
Let's delete support for pulling init actions from GitHub

Reply:
Can we force an update to the public GCS bucket? Some of the tests here rely on the updates made to the RAPIDS init action last week.